src/main/java/com/google/sps/servlets/utils/Range.java

```diff
@@ -18,7 +18,6 @@
 
 /** Range for the start and end point of sentiment scores. */
 public class Range {
-  // TODO: change the double to BigDecimal or int type to avoid imprecision in floating points
```

Comment:
seems like this has not been updated in previous PR. Can I do it here?

Reply:
I am ok with this.